### DISCUSSION TITLE
Can I measure length of linestrings and area of polygons on a panoramic image using turf?

### DISCUSSION THREAD

**Discussion by juanbrk** [Turfjs/turf#2709] [Q&A] [answered]
I work on a project where we navigate different poses and each pose has a different panoramic image. Here we can measure points, lines and polygons, and am developing a solution to display the `length` of a line and `area` of a polygon on the fly, while measurement is happening. And just wanted to know if turf is of any help on this scenario. 

**Some context**: The points of the linestring and polygon have the following interface `[{x: number, y: number}] ` and are spherical coordinates, not necesarily `latLong` coordinates.

**Comment by twelch** [comment, marked as answer]
Thanks for asking. I'm aware that at least one person uses some Turf functions for working in cartesian coordinates/space as you are describing such as a flat image.  But it is not designed or guaranteed to work properly for that.  Turf is designed for GeoJSON input, which is WGS84 lat/long coordinates.
